utensor_cgen/parser/_pbparser_impl.py

```diff
@@ -107,9 +108,7 @@ def _parse_graph_info(graph_def):
       output_content = {}
       op_attr = node.attr
       if node.op in ["Const"]:
-        for tensor_name, _, _ in output_tensor:
-          tensor = graph.get_tensor_by_name(tensor_name)
-          output_content[tensor_name] = tensor.eval()
+        output_content[tensor_name] = tensor_util.MakeNdarray(node.attr['value'].tensor)
```

Comment:
You goes too far XD
You still need to loop over `output_tensor`.
Now the `tensor_name` is undefined. LOL